src/components/Dropdown/DropdownItem.vue

```diff
@@ -66,6 +87,14 @@ export default {
     },
     getOffsetTop () {
       return this.$refs.option.offsetTop;
+    },
+    updateActiveIndex () {
+      // if the parent is DropdownItemGroup
+      if (this.$parent.group) {
+        this.$parent.updateActiveIndex(this.index);
+      } else {
+        this.$parent.activeIndex = this.index;
+      }
```

Comment:
I wasn't aware that vue had a `this.$parent` but I feel like this approach is inconsistent with our usual approach of emitting when we need to pass data/information back to the parent.  I think it's better for the child to not be coupled to the parent in this way but rather just emit something and let the parent decide what to do with/about it.